manifests/e/EclipseAdoptium/TemurinJRE/11/jdk-11.0.14+9/EclipseAdoptium.TemurinJRE.11.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: EclipseAdoptium.TemurinJRE.11
+PackageVersion: jdk-11.0.14+9
```

Version needs to be fixed
```suggestion
PackageVersion: 11.0.14.9
```